app/views/videos/search.html.haml

```diff
@@ -0,0 +1,2 @@
+- @videos.each do |video|
+	%img{:src => video.small_cover_url}
```

Comment:
The indentation looks a little bit off here - Also I would prefer `= image_tag video.small_cover_url`